pkg/es/config/config.go

```diff
@@ -220,9 +301,9 @@ func NewClient(c *Configuration, logger *zap.Logger, metricsFactory metrics.Fact
 func newElasticsearchV8(c *Configuration, logger *zap.Logger) (*esV8.Client, error) {
 	var options esV8.Config
 	options.Addresses = c.Servers
-	options.Username = c.Username
-	options.Password = c.Password
-	options.DiscoverNodesOnStart = c.Sniffer
+	options.Username = c.Authentication.BasicAuthentication.Username
+	options.Password = c.Authentication.BasicAuthentication.Password
+	options.DiscoverNodesOnStart = c.Sniffing.Enabled
```

Comment:
so for v8 only the `enabled` is used, not the UseHTTPS? Is it because v8 client always uses `https`?

Reply:
@yurishkuro yeah, that seems to be the case. The following is used when constructing the client. 
```go
// addrFromCloudID extracts the Elasticsearch URL from CloudID.
// See: https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html
func addrFromCloudID(input string) (string, error) {
	var scheme = "https://"
...
}
```

Reply:
It would be good to have a comment on the field mentioning that.

Reply:
@yurishkuro https://github.com/jaegertracing/jaeger/pull/6103